### ISSUE TITLE
Comments to javascript

### ISSUE THREAD

**Issue by GianLuca44** [VE-Glez/interactive-pricing-component-main#1]
`
let slider_range = document.querySelector('#myrange');
let switch_label = document.querySelector('.switch');
let input_checkbox = document.querySelector('.switch__checkbox');
let amount_viewers = document.querySelector('.card__viewers');
let price = document.querySelector('.card__price');

let viewers =['10K','50K','100K','500k','1M']

prices = [8.00,12.00,16.00,24.00,36.00]
`
Hi
You could use **const** instead of **let**, if these values never change.

**Comment by veglezMX** [comment]
Yes you're right, i was pretty new in javascript back then. i will change that, thank you.
